nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -698,7 +698,7 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `drbd` out-of-tree Linux kernel driver has been added in version 9.2.7. With it the DRBD 9.x features can be used instead of the 8.x features provided by the 8.4.11 in-tree driver.
 
-- `oils-for-unix`, the oil shell's c++ version is now available. The python version is still available as `oil`.
+- `oils-for-unix` is now the attribute name for the oil shell's c++ version. The python version is still available as `oil`.
```

Comment:
Sorry, I think your one is better, you can revert it if you wish...

Reply:
OK, I've removed it by force-pushing.

I was a bit confused by having an email notification about the suggestion, but not seeing it anywhere on GH: I guess you removed it before I started looking for it. (And then I re-created it myself.) Anyway, it's gone now.